lib/utils/Prompt.js

```diff
@@ -0,0 +1,10 @@
+class Prompt {
+  constructor (name, type, message, validate) {
+    this.name = name;
+    this.type = type;
+    this.message = message;
+    this.validate = validate;
+  }
```

Comment:
we discussed about another method to perform dynamic addition of <key,value> to Prompt object. The method can have the following signature.

```
add (key, value)
```